ang/civicase/PanelQuery.html

```diff
@@ -13,9 +13,9 @@ <h2 class="panel-title pull-left">
       <div class="pull-right" ng-transclude="actions"></div>
     </div>
   </header>
-  <div class="panel-body">
-    <div ng-hide="loading" ng-transclude="results"></div>
-    <div ng-if="loading">
+  <div class="panel-body" style="transition: opacity 0.2s linear;" ng-style="loading.partial && { 'opacity': 0.7 }">
```

Comment:
cant we use classes?

Reply:
i honestly wouldn't know to what component to add it. this doesn't seem to belong to `.panel-secondary`, as it's only related to the functionality of the panel-query directive, and the directive doesn't have any style associated to it nor a dedicated partial. For this reasons I thought it would be better to keep it simple and have it as an inline style (we have them already in the extension anyway, so i'm not introducing a new pattern)